src/main/java/animals/Animals.java

```diff
@@ -26,21 +26,17 @@ public void setTypeOfFood(String typeOfFood) {
         this.typeOfFood = typeOfFood;
     }
 
-
-
-
     public String toString() {
```

Comment:
Usually toString() implementation is annotated with @Override as this method is inherited from Java Object class